lib/msal-core/package.json

```diff
@@ -29,7 +29,7 @@
     "doc": "typedoc --out ./docs ./src/ --gitRevision dev",
     "build:modules": "tsc && tsc -m es6 --outDir lib-es6",
     "build": "npm run clean && npm run doc && npm run build:modules && npm run lint && npm run build:webpack && npm test",
-    "test": "karma start && jasmine tests/MSALSpec.js",
+    "test": "karma start",
```

Comment:
I know this change is made because the tests are duplicating but travis will still fail for this. :)

Reply:
Let me take a look and try to fix. thanks.

Reply:
You can also change travis config if you are sure the tests are duplicating.

Reply:
@sameerag managed to fix it somehow, thanks!